p2p/libp2p/netMessenger_test.go

```diff
@@ -866,6 +877,31 @@ func TestLibp2pMessenger_ConnectedPeersOnTopicTwoTopicsShouldWork(t *testing.T)
 	_ = mes4.Close()
 }
 
+//------- ConnectedFullHistoryPeersOnTopic
+
+func TestLibp2pMessenger_ConnectedFullHistoryPeersOnTopicShouldWork(t *testing.T) {
+	_, mes1, mes2 := createMockNetworkOf2()
```

Comment:
In order to have a better test for this function I suggest to create a special function called `createMockNetworkOf3` that will connect 3 peers out of which only 2 are full history nodes.

Reply:
Done